content/docs/studio/teams.md

```diff
@@ -0,0 +1,73 @@
+# Teams
+
+So far, you have seen how you can create personal and public views. In this
+section, we will illustrate how you can create teams. In DVC Studio, you can
+define teams with one or more team members. The team members are also called
+collaborators, and you can assign different roles to the collaborators. The
+views that you create in your team's page will be accessible to all members of
+the team.
+
+## Roles
+
+Team members can have different roles.
+
+- **View.** Users with the View access cannot edit team settings or create new
```

Comment:
This should probably be moved down after or even merged with **Invite collaborators**.

Reply:
Added to new ticket https://github.com/iterative/dvc.org/issues/2468